bindings/core/tests/combined.rs

```diff
@@ -0,0 +1,164 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use iota_sdk::{
+    client::{api::GetAddressesBuilderOptions, constants::SHIMMER_COIN_TYPE, secret::SecretManagerDto, ClientBuilder},
+    wallet::account::types::AccountIdentifier,
+};
+use iota_sdk_bindings_core::{
+    call_client_method, call_wallet_method, method_handler::Result, AccountMethod, ClientMethod, Response,
+    WalletMethod, WalletOptions,
+};
+
+#[tokio::test]
+async fn generate_addresses() -> Result<()> {
+    let client_config = r#"{
+            "nodes":[],
+            "localPow":true,
+            "fallbackToLocalPow": true
+    }"#
+    .to_string();
+
+    let client = ClientBuilder::new().from_json(&client_config)?.finish()?;
+
+    let secret_manager = format!(
+        "{{\"mnemonic\":\"{}\"}}",
+        "endorse answer radar about source reunion marriage tag sausage weekend frost daring base attack because joke dream slender leisure group reason prepare broken river"
+    );
+    let options = GetAddressesBuilderOptions {
+        coin_type: None,
+        account_index: None,
+        range: Some(std::ops::Range { start: 0, end: 10 }),
```

Comment:
fyi you can type `(3..5) instead of `std::ops::{ 0 10}`

Reply:
True, just copied and don't know who made it like this in the first place :joy: